pages/linux/auracle.md

```diff
@@ -0,0 +1,20 @@
+# auracle
+
+> Auracle is a command line tool used to interact with Arch Linux's User Repository, commonly referred to as the AUR.
+> More information: <https://github.com/falconindy/auracle>.
+
+- Find packages in the AUR by regular expression:
+
+`auracle search {{regexp}}`
```

Comment:
If I search for `*-git`, the argument should be between quotes `"`, so:
```suggestion
`auracle search "{{regex}}"`
```

Reply:
I'd suggest single quotes here if that's a regex - they play nicer with special characters such as `$`.